### ISSUE TITLE
 ERROR: Could not create SDL2 window: Failed loading libGL.so.1: libGL.so.1: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by marcellodesales** [pinfake/android-react-native#2]
Thanks for making this... Super clean Docker-compose envs for Emulator and Development... That's what I was looking for... 

Well, I'm trying to run and I'm getting this error... Could you help!?!?

```
mdesales@kauai:~/dev/github/public/pinfake/android-react-native$ docker-compose up -d emulator
Creating androidreactnative_emulator_1


mdesales@kauai:~/dev/github/public/pinfake/android-react-native$ docker-compose logs
Attaching to androidreactnative_emulator_1
emulator_1     | emulator: WARNING: Increasing RAM size to 1GB
emulator_1     | emulator: warning: opening audio output failed
emulator_1     | 
emulator_1     | ERROR: Could not create SDL2 window: Failed loading libGL.so.1: libGL.so.1: cannot open shared object file: No such file or directory
emulator_1     | 
```

Docker Info
====

My OS is ubuntu 16 64bits... Thanks a lot!

```
mdesales@kauai:~/dev/github/public/pinfake/android-react-native$ docker info
Containers: 1
 Running: 0
 Paused: 0
 Stopped: 1
Images: 2
Server Version: 17.03.0-ce
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 13
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 977c511eda0925a723debdc94d09459af49d082a
runc version: a01dafd48bc1c7cc12bdb01206f9fea7dd6feb70
init version: 949e6fa
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.4.0-31-generic
Operating System: Ubuntu 16.04.1 LTS
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 1.937 GiB
Name: kauai
ID: HFB6:MYBM:HEN2:ZCPS:GRNU:YQVS:ND7Q:HGDC:7Z63:6NAO:NYUT:B5KO
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Comment by pinfake** [comment]
https://github.com/pinfake/android-emulator-x11/issues/2

**Comment by marcellodesales** [comment]
Fixed with pinfake/android-emulator-x11#2
